admin/make_homebrew_recipe.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Create a Homebrew recipe for Flocker, using the VERSION environment variable.
+
+Invoke using e.g.:
+  $ export VERSION=0.1.2
+  $ ./make_homebrew_recipe.py > flocker-${VERSION}.rb
+
+Inspired by https://github.com/tdsmith/labmisc/blob/master/mkpydeps.
+"""
+
+from os import environ
+from json import load
+from urllib2 import urlopen
+from hashlib import sha1
+from tl.eggdeps.graph import Graph
+
+version = environ.get("VERSION")
+if version is None:
+    raise Exception("Set the VERSION environment variable.")
+
+url = ("http://storage.googleapis.com/archive.clusterhq.com/"
+       "downloads/flocker/Flocker-{version}.tar.gz").format(version=version)
+tarball = urlopen(url)
+checksum = sha1(tarball.read()).hexdigest()
+tarball.close()
+
+class_name = 'Flocker' + version
+for disallowed_character in ['-', '.']:
+    class_name = class_name.replace(disallowed_character, '')
+
+resources = []
+graph = Graph()
+graph.from_specifications(["flocker"])
+dependencies = []
+for name, node in sorted(graph.iteritems()):
+    if name == "flocker":
+        continue
+
+    requirement = node.dist.as_requirement()
+    operator, version = requirement.specs[0]
+    project_name = requirement.project_name
+    dependencies.append(project_name)
+    url = "http://pypi.python.org/pypi/{name}/{version}/json".format(
+            name=project_name,
+            version=version)
```

Comment:
pyflakes says `make_homebrew_recipe.py:48:13: E126 continuation line over-indented for hanging indent`